src/plugins/intel_cpu/src/graph_optimizer.cpp

```diff
@@ -561,78 +561,78 @@ void GraphOptimizer::FuseConvolutionAndZeroPoints(Graph &graph) {
+            return false;
+        }
 
-                for (int i = 2; i < zpDims.size(); i++) {
-                    if (zpDims[i] != 1)
-                        return false;
-                }
+        auto zpDims = parent0->getInputShapeAtPort(1).getDims();
+        if (zpDims[0] != 1 || !dimsEqualStrong(zpDims[1], IC))
+            return false;
 
-                auto arg1 = parent0->getParentEdgesAtPort(0)[0]->getParent();
-                if (arg1->getOriginalOutputPrecisionAtPort(0) != Precision::U8)
-                    return false;
+        for (int i = 2; i < zpDims.size(); i++) {
+            if (zpDims[i] != 1)
+                return false;
+        }
 
-                auto zeroPointsConstant = dynamic_cast<node::Input*>(arg0.get());
-                if (zeroPointsConstant == nullptr)
-                    IE_THROW() << "Cannot cast to Input node";
+        auto arg1 = parent0->getParentEdgesAtPort(0)[0]->getParent();
+        if (arg1->getOriginalOutputPrecisionAtPort(0) != Precision::U8)
+            return false;
 
-                auto zeroPointsBlob = zeroPointsConstant->getMemoryPtr();
-                if (zeroPointsBlob == nullptr)
-                    IE_THROW() << "Cannot cast to TBlob internal zero points blob";
+        auto zeroPointsConstant = dynamic_cast<node::Input*>(arg0.get());
+        if (zeroPointsConstant == nullptr)
+            IE_THROW() << "Cannot cast to Input node";
 
-                auto zeroPointsData = static_cast<const uint8_t*>(zeroPointsBlob->GetPtr());
-                if (zeroPointsData == nullptr)
-                    IE_THROW() << "zeroPointsBlob has not allocated buffer";
+        auto zeroPointsBlob = zeroPointsConstant->getMemoryPtr();
+        if (zeroPointsBlob == nullptr)
+            IE_THROW() << "Cannot cast to TBlob internal zero points blob";
 
-                auto zeroPointDataSize =  parent0->getInputShapeAtPort(1).getDims()[1];
-                if (Shape::UNDEFINED_DIM == zeroPointDataSize) {
-                    return false;
-                }
+        auto zeroPointsData = static_cast<const uint8_t*>(zeroPointsBlob->GetPtr());
+        if (zeroPointsData == nullptr)
+            IE_THROW() << "zeroPointsBlob has not allocated buffer";
 
-                for (int j = 0; j < zeroPointDataSize; j++) {
-                    convNode->inputZeroPoints.push_back(zeroPointsData[j]);
-                }
-            } else {
-                return false;
-            }
-        } else {
+        auto zeroPointDataSize =  parent0->getInputShapeAtPort(1).getDims()[1];
+        if (Shape::UNDEFINED_DIM == zeroPointDataSize) {
             return false;
         }
+        auto zeroPointEqualCnt = 0;
+        for (int j = 0; j < zeroPointDataSize; j++) {
+            convNode->legacyInputZeroPoints.push_back(zeroPointsData[j]);
+            if (zeroPointsData[j] == zeroPointsData[0])
+                zeroPointEqualCnt++;
+        }
+        //check whether zeropoint is per-tensor.
```

Comment:
I meant a bit different comment.
As far as I understand we prepare two types of zeropoints here and 
we let the node to decide later, which one should be actually used.
Can we have the similar explanation in the comment?